pkg/kubelet/volumemanager/volume_manager.go

```diff
@@ -124,7 +124,7 @@ type VolumeManager interface {
 	// is added to the desired state of world, indicating it *should* be
```

Comment:
Shouldn't this comment be made more precise to say that the volume implements the Attacher or DeviceMounter interfaces?

Reply:
Done